src/main/java/com/uid2/shared/attest/UidCoreClient.java

```diff
@@ -94,44 +94,43 @@ public ICloudStorage getContentStorage() {
         return this.contentStorage;
     }
 
-    public void attest() throws IOException, UidCoreClientException {
-        attestInternal();
-    }
-
+    /**
+     * HTTP Get with automatic attestation on first 401 Unauthorized response
+     * @param url API endpoint
+     * @return InputStream from HTTP response
+     * @throws IOException on HTTP failures
+     * @throws UidCoreClientException on attestation failures
+     */
     private InputStream getWithAttest(String url) throws IOException, UidCoreClientException {
-        if (!attested())
-            attestInternal();
-
-        URLConnection conn = sendGet(url);
-
-        if (conn instanceof HttpURLConnection && attestIfRequired((HttpURLConnection) conn))
```

Comment:
I think this was preventing attestation in standalone dev mode. Does the standalone mode work now? What prevents attestation path in that mode?